projects/cdk/schematics/ng-update/constants/enums.ts

```diff
@@ -91,7 +91,7 @@ export const ENUMS_TO_REPLACE: ReplacementEnum[] = [
             Error: 'error',
             Pass: 'pass',
         },
-        keepAsType: false,
+        keepAsType: true,
```

Comment:
We have 
https://github.com/Tinkoff/taiga-ui/blob/892e1770b40ad47c89d83d2484eeb7cfa34bcd2d/projects/kit/types/step-state.ts#L1